pvlib/test/test_atmosphere.py

```diff
@@ -113,3 +114,40 @@ def test_first_solar_spectral_correction_supplied():
 def test_first_solar_spectral_correction_ambiguous():
     with pytest.raises(TypeError):
         atmosphere.first_solar_spectral_correction(1, 1)
+
+
+def test_kasten96_lt():
+    """Test Linke turbidity factor calculated from AOD, Pwat and AM"""
+    atmos_path = os.path.dirname(os.path.abspath(__file__))
+    pvlib_path = os.path.dirname(atmos_path)
+    melbourne_fl = tmy.readtmy3(os.path.join(
+        pvlib_path, 'data', '722040TYA.CSV')
```

Comment:
no particular reason, I like that Melbourne-FL site, it's close to Cocoa where I have irradiance data. also although no site matches the Linke turbidity data well, the Alaska site has bigger differences, and also bigger differences between broadband approximations relative to Melbourne-FL. Maybe these are not good tests?